rfc/0052-sdk3-full-text-search.md

```diff
@@ -635,6 +635,7 @@ Important SDK considerations:
   * The `k` parameter is mandatory.  The FTS team suggest 3 is a sensible default to send from the SDKs.
```

Comment:
If there are sensible defaults that could be sent, what is the reasoning behind them being required fields in the first place rather than having the server specify the defaults if they are not specified?  I think this is something we've been discussing in other contexts, where the SDKs should attempt to avoid specifying concrete 'default' values.